test/utils/ReliefUtilsSpec.scala

```diff
@@ -173,16 +173,25 @@ class ReliefUtilsSpec extends PlaySpec with ReliefConstants {
     }
 
     "Return Property trading, if propertyTrading is true" in {
-      implicit val hc = new HeaderCarrier(sessionId = Some(SessionId(s"session-${UUID.randomUUID}")))
+      implicit val hc = HeaderCarrier(sessionId = Some(SessionId(s"session-${UUID.randomUUID}")))
 
       val reliefs = ReliefBuilder.reliefTaxAvoidance(atedRefNo, periodKey, Reliefs(periodKey, propertyTrading = true))
       val reliefsRequest = ReliefUtils.convertDraftReliefsToDraftDescription(reliefs)
 
       reliefsRequest must be(Seq("Property trading"))
     }
 
+    "Return equity release, if equity release is true" in {
```

Comment:
Would it be worth adding the additional tests for Social Housing, Open to the public, Lending etc to completely be confident ?

Reply:
Yeah, but under a different ticket. I think its better to to get this bug fix out to prod since if a user was to see it-it would be look bad.